internal/resources/zero_trust_tunnel_cloudflared_route/v4_to_v5.go

```
@@ -0,0 +1,128 @@
+	result := instance.String()
+
+	// Set schema_version to 0 for v5
+	result, _ = sjson.Set(result, "schema_version", 0)
+
+	// Update the type field if it exists (for unit tests that pass instance-level type)
+	if instance.Get("type").Exists() {
+		result, _ = sjson.Set(result, "type", "cloudflare_zero_trust_tunnel_cloudflared_route")
+	}
+
+	// Try to update the ID if API credentials are available
+	// In v4, the ID was the network CIDR (or a checksum of it)
+	// In v5, the ID is a UUID from the API
+	attrs := instance.Get("attributes")
+	if attrs.Exists() {
+		currentID := attrs.Get("id").String()
+		accountID := attrs.Get("account_id").String()
+		tunnelID := attrs.Get("tunnel_id").String()
+		network := attrs.Get("network").String()
+		virtualNetworkID := attrs.Get("virtual_network_id").String()
+
+		// Check if the ID looks like a UUID (36 chars with dashes at positions 8, 13, 18, 23)
+		// If it already is a UUID, no need to update
+		if len(currentID) == 36 && currentID[8] == '-' && currentID[13] == '-' && currentID[18] == '-' && currentID[23] == '-' {
+			return result, nil
+		}
+
+		// Try to query the API if credentials are available
+		apiToken := os.Getenv("CLOUDFLARE_API_TOKEN")
+		apiKey := os.Getenv("CLOUDFLARE_API_KEY")
+		apiEmail := os.Getenv("CLOUDFLARE_EMAIL")
+
+		if apiToken != "" || (apiKey != "" && apiEmail != "") {
+			// Create Cloudflare client using v6 API
+			// NewClient reads credentials from environment variables:
+			// CLOUDFLARE_API_TOKEN or (CLOUDFLARE_API_KEY + CLOUDFLARE_EMAIL)
+			client := cloudflare.NewClient()
+
+			// Query for the tunnel route using v6 API
+			params := zero_trust.NetworkRouteListParams{
+				AccountID: cloudflare.F(accountID),
+				IsDeleted: cloudflare.F(false),
+				TunnelID:  cloudflare.F(tunnelID),
+			}
+			if virtualNetworkID != "" {
+				params.VirtualNetworkID = cloudflare.F(virtualNetworkID)
+			}
+
+			page, err := client.ZeroTrust.Networks.Routes.List(context.Background(), params)
```

Also here if the err is not nil, its swallowed. I think we need to add a diagnostic or logging.